ci/Jenkinsfile.combined

```diff
@@ -0,0 +1,115 @@
+pipeline {
+  agent { label 'master' }
+
+  options {
+    disableConcurrentBuilds()
+    buildDiscarder(logRotator(
+      numToKeepStr: '10',
+      daysToKeepStr: '30',
+      artifactNumToKeepStr: '10',
+    ))
+  }
+  
+  stages {
+    stage('Tag') {
+      steps { script {
+        common = load('ci/common.groovy')
+        /* to avoid race conditions in parallel builds */
+        print "Build Number: ${common.tagBuild()}"
+      } }
+    }
+    stage('Build') {
+      parallel {
+        stage('MacOS') {
+          steps { script {
+            osx = build('status-react/combined/desktop-macos')
+          } }
+        }
+        stage('Linux') {
+          steps { script {
+            nix = build('status-react/combined/desktop-linux')
+          } }
+        }
+        stage('iOS') {
+          steps { script {
+            ios = build('status-react/combined/mobile-ios')
+          } }
+        }
+        stage('Android') {
+          steps { script {
+            dro = build('status-react/combined/mobile-android')
+          } }
+        }
+        stage('Android e2e') {
```

Comment:
It looks like a mix of build stages and one stage running e2e tests. Why is it so?

Reply:
Because Android e2e build is different than the regular stage?

Reply:
Ah ok. I did not know that.